### PR TITLE
Add work-about sections

### PR DESCRIPTION
**Pull request by Behnam1369**
Add work section.
Add about section.
Modify .gitignore file.

### CODE REVIEW

**Review by leolpaz — COMMENTED** [review]
Hi @Behnam1369 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain a short summary of what was added in the PR description
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -25,6 +25,110 @@ <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
...
+                    <span class="span2">Full Stack Developer</span>
+                    <i class="devider"></i>
+                    <span class="span2">2018</span>
+                    <p>
+                        A web application which help people to find partners for thair sport activities. 
+                        You can simply search for partners, get connected to them and enjoy your sport activities together.
+                    </p>
+                    <span class="tag">html</span>
+                    <span class="tag">css</span>
+                    <span class="tag">javascript</span>
```
Comment (Lordkaito) [inline index.html]: - [ ] Please, make sure to use `<ul>` tags for your languages here and the correspondent `<li>` tag as well. This is really helpful to make your code more semantic, so later on, whenever you (or any other developer) see your project you can understand it without struggling.

File: index.html [hunk 2]
```diff
@@ -25,6 +25,110 @@ <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
...
+                </div>
+            </section>
+            <div class="seprator2"></div>
+            <section id="about">
+                <h2>About me</h2>
+                <p>I am a professional web developer. I have been working as a software developer since 2016. 
+                You can find a couple of my works here. If you like what you see here feel free to contact me. 
+                I will be happy if I can help you with your software needs. 
+                </p>
+                <span >Follow me on social media</span>
```
Comment (Lordkaito) [inline index.html]: - [ ] Kindly use `<hx>` tags for subtitles like this one. As I mentioned earlier, this helps your code to be more semantic.

File: index.html [hunk 3]
```diff
@@ -25,6 +25,110 @@ <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
...
+                <i class="fa fa-chevron-down"></i>
+                <ul class="tech">
+                    <li class="javascript">Javascript</li>
+                    <li class="html">HTML</li>
+                    <li class="css">CSS</li>
+                </ul>
+                <h3>Frameworks</h3>
+                <i class="fa fa-chevron-right"></i>
+                <hr/>
+                <h3>Skills</h3>
```
Comment (Lordkaito) [inline index.html]: - [ ] Please change these `<h3>` tags that you used for "Languages", "Frameworks" and "Skills" into a `<li>` tags, and remember to put them all together into a `<ul>`. The result should be a nested `<ul>` (one `<ul>` inside another `<ul>`) between "Languages" and "JavaScript", "HTML" and "CSS", for example.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Behnam1369, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Behnam1369** [comment]
- Use list tags instead of `span` for tag class
- Use `h5` instead of `span` for subtitle
- Use nested lists in the technologies used section.

**Review by ReshyResh — APPROVED** [review]
# Hi @Behnam1369 , 

### You did a wonderful job implementing the requested changes!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**